pkg/agent/openflow/pipeline.go

```diff
@@ -1575,6 +1634,17 @@ func (c *client) serviceEndpointGroup(groupID binding.GroupIDType, withSessionAf
 	return group
 }
 
+func (c *client) getNodeIPProtocols() []binding.Protocol {
```

Comment:
if this could called many times, especially per policy, better to have a static slice in client and set it when initializing to avoid unnecessary memory allocation and release.